### ISSUE TITLE
Orientation when soldering

### ISSUE THREAD

**Issue by jonathf** [PJE66/hummingbird#3]
Is the keyboard reversable? If not, can I assume the print side is up?

**Comment by safstromo** [comment]
Ive just built this, everything is mounted on the same side, print side is up. ![PXL_20231230_120017811.MP.jpg](https://github.com/PJE66/hummingbird/assets/56272436/d11b323b-1a67-4348-b719-775ddd568cd8)

**Comment by jonathf** [comment]
Thanks for the feedback. Good to know. I just recieved the parts for the keyboard and is about to start assembly.

Am I seeing right that your controller is soldered directly on top of the pcb? I ask because I have millmax low profile sockets, but I am realizing that even they seem to be a bit too tall for the 3d-printed case parts.

**Comment by safstromo** [comment]
Yeah i just put it directy since i wanted to keep the low profile, might comeback to bite me if the controller break :smile:
